pkg/runtime/handler.go

```diff
@@ -101,19 +101,45 @@ func (h *handlerFactory) Build(handler *pb.Handler, instances []*pb.Instance, en
 	hndlrBldrInfo, _ := h.builderInfoFinder(handler.Adapter)
 
 	hndlrBldr := hndlrBldrInfo.NewBuilder()
+
```

Comment:
Nit: I find hndlrbuilder name too long and redundant.
You can drop hndlr prefix here. We are already in handler.go file and there is only 1 thing to build.

Similarly handlerbuilderinfo can just be info

Reply:
D